rmc/utils/generic.py

```diff
@@ -163,78 +122,78 @@ def annotate_and_filter_codons(ht: hl.Table) -> hl.Table:
 ## Reference genome processing-related utils
 ####################################################################################
 def process_context_ht(
+    filter_to_canonical: bool = False,
     filter_csq: Set[str] = None,
-    filter_outlier_transcripts: bool = False,
```

Comment:
I removed this because we might not want to remove outlier transcripts anymore, right? one of the requests we got with v2 was to release all canonical transcripts (outlier or not)

Reply:
I was thinking the same but mentioned it in case non-outliers are in the logic for functions that use `process_context_ht` - I'm looking at `get_aa_from_context` and `prepare_amino_acid_ht`

Reply:
`get_aa_from_context` still supports transcript filtering and I haven't touched `prepare_amino_acid_ht` since that's in `missense_badness.py` (I've only fixed import errors in that script)

Reply:
Ok!

Reply:
Let's just remove "non-outlier" in line 220 in `get_aa_from_context`

Reply:
I'll add a comment here https://github.com/broadinstitute/regional_missense_constraint/blob/f4490e5227959cd526c0b605d9d3c95c2165e50e/rmc/utils/missense_badness.py#L64

Reply:
And add a TODO in the missense badness function using `process_context_ht`?